### DISCUSSION TITLE
绘制组件初始化绘制面会出现一条长长的线指向左小角

### DISCUSSION THREAD

**Discussion by yadanchina** [zouyaoji/vue-cesium#224] [Q&A]
@zouyaoji 
绘制组件初始化绘制面会出现一条长长的线指向左小角,不停的闪烁，将地图切换成2d的尤其明显，还会导致掉帧

**Comment by zouyaoji** [comment]
在线文档能重现你说的这个现象吗？

**Reply by yadanchina** [reply]
在线文档，暂时没有复现
我只加载
![image](https://user-images.githubusercontent.com/42006019/178882140-27b2a530-591e-4d84-8255-584cc990e216.png)
这几个组件

**Reply by yadanchina** [reply]
而且绘制时会出现页面高度宽度莫名增加，出现上下滚动条

**Comment by yadanchina** [comment]
![image](https://user-images.githubusercontent.com/42006019/178881483-6912673f-f891-4048-a47e-8f8b26c7aebc.png)

@zouyaoji 
这个是实际出现的效果

**Reply by zouyaoji** [reply]
没能重现，你看看升级一下你vue-cesium的版本试试？

**Reply by yadanchina** [reply]
已经是最新版本了

**Reply by zouyaoji** [reply]
你弄个能重现的工程传上来吧。

**Reply by yadanchina** [reply]
https://codesandbox.io/s/ecstatic-rain-tr9qw8?file=/src/App.vue
@zouyaoji

**Reply by zouyaoji** [reply]
能重现了，我回头看看。

**Reply by zouyaoji** [reply]
我测了，这是cesium本身的bug，等下去提一个issue。

**Reply by zouyaoji** [reply]
https://github.com/CesiumGS/cesium/issues/10620

**Reply by yadanchina** [reply]
感谢大佬
